pkg/apis/core/types.go

```diff
@@ -3198,6 +3198,25 @@ type VolumeMountStatus struct {
 	// depending on the mount result.
 	// +optional
 	RecursiveReadOnly *RecursiveReadOnlyMode
+	// volumeStatus represents volume-type-specific status about the mounted
+	// volume.
+	VolumeStatus
```

Comment:
this file is not very consistent with marking things optional Perhaps this should be marked as such explicitly as other fields in this type

```suggestion
    // +ptional
	VolumeStatus
```